src/robot_interface/models/mission/task.py

```diff
@@ -1,11 +1,8 @@
 from dataclasses import dataclass, field
-from datetime import datetime
-from typing import Iterator, List, Optional, Union
+from typing import Iterator, List, Optional
 from uuid import UUID, uuid4
 
-from isar.apis.models.models import StartMissionResponse, TaskResponse
-from isar.config.settings import settings
-from isar.models.mission_metadata.mission_metadata import MissionMetadata
+from isar.apis.models.models import TaskResponse
```

Comment:
Same thing as we did for `StartMissionResponse`, we dont want `TaskResponse` in this file, as we dont want any imports from ISAR to avoid circular dependencies